extensions/ql-vscode/src/remote-queries/repository-selection.ts

```diff
@@ -45,9 +45,17 @@ export async function getRepositorySelection(
         case DbItemKind.RemoteSystemDefinedList:
           return { repositoryLists: [selectedDbItem.listName] };
         case DbItemKind.RemoteUserDefinedList:
-          return {
-            repositories: selectedDbItem.repos.map((repo) => repo.repoFullName),
-          };
+          if (selectedDbItem.repos.length === 0) {
+            throw new Error(
```

Comment:
I think it's better to do `showAndLogErrorMessage` rather than throwing an exception. It has the same outcome from a user's perspective but it won't be considered as an unhandled error from our app's perspective.

Reply:
Good point, I changed the other errors in that switch as well!